src/layouts/RabitLayout/Sidebar/index.tsx

```diff
@@ -67,12 +66,12 @@ function Sidebar() {
                     {/* rabit logo is created here */}
                     <Logo />
                 </TopSection>
-
+                
+                {/* upload button created here
                 <Fab variant='extended' color='primary' onClick={() => navigate('/upload')}>
-                    {/* upload button created here */}
                     <UploadIcon sx={{ mr: 1 }} /> Upload
-                </Fab>
-
+                </Fab>*/}
```

Comment:
If it's a permanent decision to remove the previous upload button (a good idea I think), then please remove this commented out code

Reply:
done, removed unused imports also